webpack/ForemanTasks/Components/TaskDetails/TasksDetailsHelper.js

```diff
@@ -1 +1 @@
-export const getTaskID = () => window.location.pathname.split('/').slice(-1)[0];
+export const getTaskID = () => window.location.pathname.split('/').filter(i => i).slice(-1)[0];
```

Comment:
Could you add a comment why it is there? Maybe it would be obvious to js folks, but it took me a while to realize why filtering by identity would help